In-house patch
makes the original "00-foomatic-kitload.in.patch" more general
...to be promote-able to comunity.

--- a/foomatic-kitload.in	2016-07-20 00:40:31.000000000 -0700
+++ b/foomatic-kitload.in	2016-07-20 13:48:17.059703347 -0700
@@ -54,7 +54,7 @@
 $ENV{'LANG'} = "C";
 
 # Copy the files
-my $retval = system("( cd $dir ; tar cf - --exclude CVS . ) | tar xv${dontoverwrite}Cf $destdir$libdir/db/source - 2> $destdir$libdir/kitload2.log | tee $destdir$libdir/kitload.log");
+my $retval = system("( cd $dir ; @TAR@ cf - --exclude CVS . ) | @TAR@ xv${dontoverwrite}Cf $destdir$libdir/db/source - 2> $destdir$libdir/kitload2.log | tee $destdir$libdir/kitload.log");
 #$retval = $retval / 256;
 #if ($retval) {
 #    die "Error copying files to $destdir$libdir/db/source: $?";
